1. setuptools no longer supports use_2to3, so drop the setup() arg
2. Drop the pystache-test entrypoint script here rather than in-template

--- ./setup.py
+++ ./setup.py
@@ -351,13 +351,6 @@
 
     """
     extra = {}
-    # TODO: it might be more correct to check whether we are using
-    #   Distribute instead of setuptools, since use_2to3 doesn't take
-    #   effect when using Python 2, even when using Distribute.
-    if py_version >= (3, ):
-        # Causes 2to3 to be run during the build step.
-        extra['use_2to3'] = True
-
     return extra
 
 
@@ -401,7 +394,6 @@
           entry_points = {
             'console_scripts': [
                 'pystache=pystache.commands.render:main',
-                'pystache-test=pystache.commands.test:main',
             ],
           },
           classifiers = CLASSIFIERS,
